src/loaders/HubsTextureLoader.js

```diff
@@ -52,13 +25,39 @@ export default class HubsTextureLoader {
     return texture;
   }
 
-  setCrossOrigin(value) {
-    this.crossOrigin = value;
-    return this;
+  async loadAsync(url, onProgress) {
```

Comment:
Oh hah, I mentioned this earlier in the review. Apparently we already have it :)